pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -166,6 +173,7 @@ func TestWatch(t *testing.T) {
 }
 
 func TestUpdateStatus(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PersistentVolumeLastPhaseTransitionTime, true)()
```

Comment:
Should we add more testcases? i.e.  one's featute-gate is on and another's featute-gate is off.

Reply:
Tests were added mostly in alpha to `strategy_test.go` and are quite extensive, pretty much covering all possible scenarios in combination with feature gate enablement/disablement. I just fixed this test because timestamp addition broke it (as expected). Is there some specific test case that you have in mind that we might have missed?